kmk/modules/trilayerstate.py

```diff
@@ -0,0 +1,211 @@
+'''One layer isn't enough. Adds keys to get to more of them'''

```

Comment:
Would be nice to see unit tests for this new module. At the very least, it'd illustrate how it's supposed to work.